go/go-gin-mysql-collector/app/main.go

```diff
@@ -0,0 +1,166 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gin-gonic/gin"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/XSAM/otelsql"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+	"go.opentelemetry.io/otel/exporters/stdout/stdouttrace"
+	"go.opentelemetry.io/otel/propagation"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.7.0"
+)
+
+var tracer = otel.Tracer("opentelemetry-go-gin")
+
+func newConsoleExporter() (sdktrace.SpanExporter, error) {
+	return stdouttrace.New(
+		stdouttrace.WithWriter(os.Stdout),
+		stdouttrace.WithPrettyPrint(),
+		stdouttrace.WithoutTimestamps(),
+	)
+}
+
+func initTracer() func(context.Context) error {
+	client := otlptracehttp.NewClient(
+		otlptracehttp.WithInsecure(),
+		otlptracehttp.WithEndpoint("appsignal:8099"),
```

Comment:
```suggestion
		otlptracehttp.WithEndpoint("appsignal-collector:8099"),
```
Let's use the same name as in other apps.